src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,31 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from "../CommentList";
+import './PostInfo.scss';
+import { UserInfo } from "../UserInfo";
+
+export const PostInfo = ({ post }) => (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {post.title}
+        </h3>
+        <p>
```

Comment:
Ensure that the HTML structure and semantics are correct. It's recommended to wrap the 'Posted by ' text and the UserInfo component within the same HTML element for better readability and semantics. You could use a span or div here instead.